src/popups/PopupSongNotes.as

```diff
@@ -162,6 +163,9 @@ package popups
 
             optionMusicOffset = new ValidatedText(box, xOff, yOff, 100, 20, ValidatedText.R_FLOAT, changeHandler);
             optionMusicOffset.text = "0";
+
+            //- Apply Current Offsets
+            populateOffsets = new BoxButton(box, xOff + 110, yOff, 180, 27, "Apply Current Offsets", 12, clickHandler);
```

Comment:
Would it be possible to add a string translation ID for `Apply Current Offsets`?

Reply:
Added.